pkg/roachprod/install/cockroach.go

```diff
@@ -937,6 +952,62 @@ func (c *SyncedCluster) initializeCluster(
 	return res, err
 }
 
+// createAdminUserForSecureCluster creates a `roach` user with admin
+// privileges. The password used matches the virtual cluster name
+// ('system' for the storage cluster).
+func (c *SyncedCluster) createAdminUserForSecureCluster(
+	ctx context.Context, l *logger.Logger, startOpts StartOpts,
+) (*RunResultDetails, error) {
+	if !c.Secure {
+		return &RunResultDetails{}, nil
+	}
+
+	const username = "roach"
+	// N.B.: we cannot reuse username and password combinations across
```

Comment:
nit: we _can_ use the same username and password, but it leads to weird behavior, right?

Reply:
Reworded the comment here. We _can_, but then the virtual cluster console becomes impossible to access.